File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        r = 0
        w = 0
        b = 0

        for i in range(0, len(nums)):
            if nums[i] == 0:
                r += 1
            elif nums[i] == 1:
                w += 1
            else: 
                b += 1

        nums[:r] = [0] * r
        nums[r:r+w] = [1] * w
        nums[r+w:] = [2] * b

        return nums

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int r = 0, w = 0, b = 0;
        for (int i = 0; i < nums.size(); ++i) {
            if (nums[i] == 0) ++r;
            else if (nums[i] == 1) ++w;
            else ++b; 
        };
        nums.erase(nums.begin(), nums.begin() + r);
        nums.insert(nums.begin(), r, 0);
        nums.erase(nums.begin()+r, nums.begin() + r + w);
        nums.insert(nums.begin() + r, w, 1);
        nums.erase(nums.begin() + r + w, nums.end());
        nums.insert(nums.begin() + r + w, b, 2);
    };
};